### PR TITLE
MAINTAINERS: add @karthik-script as collaborator

### PR DESCRIPTION
**Pull request by karthik-script**
Closes https://github.com/tldr-pages/tldr/issues/14539

- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, can you check if you have accepted the invitation you received in the mail since the annotation in the CODEOWNERS file mentions that you don't have write access to the repo?

image

**Comment by karthik-script** [comment]
@kbdharun Done. Thank you.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**
